internal/k8s/controller.go

```diff
@@ -467,6 +467,10 @@ func AddPodEventHandler(numWorkers uint32, c *AviController) cache.ResourceEvent
 				utils.AviLog.Debugf("key : %s, msg: same resource version returning", key)
 				return
 			}
+			if _, ok := pod.GetAnnotations()[lib.NPLPodAnnotation]; !ok {
```

Comment:
this code seems to be repeated everywhere, can it me moved into a common function ?

Reply:
I have addressed the comment. Thanks